### ISSUE TITLE
Spike | Investigate integrating 686/674 submission process with BPDS system

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#93959]
## Issue Description
The BPDS system is designed to accept raw json content for individual forms, which would eliminate the need for OCR to extract data from 686/674 (and PCPG and VRE) before landing in VBMS. BPDS stands for Benefits Process Data Service.

The spec can be [found here](https://bpds-dev.dev.bip.va.gov/swagger-ui.html#/). BPDS is a repository designed to house JSON versions of scanned VA forms.
The plan would be to continue with existing PDF submission process but we'd send json to BPDS in addition to existing steps. 
While ticket is for 686/674, the proposed plan / design should work for any other forms. 

These are the links to various environments that they have
- https://bpds-dev.dev.bip.va.gov/swagger-ui.html#/
- https://bpds-test.dev.bip.va.gov/swagger-ui.html#/
- https://bpds-uat.stage.bip.va.gov/swagger-ui.html#/

Before this work starts, we may need to get an approval from VA security teams. Sanja is looking into it. 

[BPDS Integration.pdf](https://github.com/user-attachments/files/16984476/BPDS.Integration.pdf)
## Tasks
- [ ] Document steps needed to integrate with BPDS
- [ ] Document steps / options to load test BPDS system. How many forms per second can they take? 
- [ ] Create tickets to the actual integration. 
- [ ] Build integration prototype (maybe this is a separate ticket?)

## Acceptance Criteria
- [ ] We have a blue print of the work needed to complete this integration. 
---
## How to configure this issue
- [ ] **Attached to an Epic** (This work will become part of an Epic, once it is ready for implementation)

**Comment by mjknight50** [comment]
Putting data from 686/674 into BPDS wouldn't be useful at the moment because there is no mechanism to pull it out of there.
